ansible/roles/k8s_master/tasks/join_master_member.yml

```diff
@@ -0,0 +1,31 @@
+- name: Generated token - 1.
+  local_action: shell sed -n 70,73p ./token > ./mastertoken
+
+- name: Copy master token
+  copy: 
+    src: ./mastertoken
+    dest: /tmp/join-command.sh
+    owner: root
+    group: root
+    mode: '0777'
+
+- name: Generated token - 2.
+  shell: sed -i 's/\b:80\b/& --apiserver-advertise-address {{ ansible_ens2.ipv4.address }} --ignore-preflight-errors=NumCPU/' /tmp/join-command.sh
```

Comment:
1. sed with hard coded line numbers seems not stable or not future proof. Could you improve?
2. No need to write temporary files, you may just manipulate the strings and construct a shell command.

Reply:
That is a good point, I removed the hard coded line numbers and stored relevant output in Ansible register/variable

Reply:
Thank you for all of the suggestions!